lib/active_model_serializers/json_api_objects/json_api.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveModelSerializers::JsonApiObjects
+  module JsonApi
```

Comment:
It adds the jsonapi hash (`object`) to whatever you pass to add!

So, I think that this module should explain why, and what the end result would look like:

e.g.:
```
{ 
  data: { ... },
  jsonapi: { ... }
}
```